src/main/java/eu/bbmri_eric/negotiator/service/UserNotificationServiceImpl.java

```diff
@@ -51,6 +75,29 @@ private static Set<Person> getRepresentativesForNegotiation(Negotiation negotiat
         .collect(Collectors.toSet());
   }
 
+  private static Set<Person> getRepresentativesForStaleNegotiation(Negotiation negotiation) {
+    Set<Person> representatives =
```

Comment:
first you should just get resources involved in the Negotiation that did not have a new record in the last week/2 week. Then simply get their reps